paddle/fluid/operators/mean_op.h

```diff
@@ -55,8 +55,8 @@ class MeanGradKernel : public framework::OpKernel<T> {
     IG->mutable_data<T>(context.GetPlace());
 
     T ig_size = static_cast<T>(IG->numel());
-    Eigen::DSizes<int, 1> bcast(ig_size);
-
+    Eigen::DSizes<int, 1> bcast(static_cast<int>(ig_size));
+    // EigenVector<T>::From can cast float to float16?
```

Comment:
seems no cast needed?

Reply:
It is necessary, otherwise the compile throw errors.